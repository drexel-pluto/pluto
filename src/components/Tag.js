import React from "react";
import { TouchableHighlight, Text, StyleSheet } from "react-native";
import { Colors, Typography, Layouts, Mixins } from "../styles/index";

export default Tag = props => {
  const { id, tagName } = props;
  return (
    <TouchableHighlight style={styles.tag}>
      <Text>{tagName}</Text>
    </TouchableHighlight>
  );
};

const styles = StyleSheet.create({
  tag: {
    paddingHorizontal: 20,
    paddingVertical: 10,
    marginRight: 20,
    backgroundColor: Colors.GRAY_LIGHT
  }
});
