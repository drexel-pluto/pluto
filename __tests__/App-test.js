import 'react-native';
import React from 'react';
import Tag from '../src/components/Tag';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Tag />).toJSON();
  expect(tree).toMatchSnapshot();
});