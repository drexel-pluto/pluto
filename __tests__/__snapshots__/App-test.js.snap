// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#dbdbdb",
      "borderRadius": 50,
      "marginRight": 40,
      "opacity": 1,
      "paddingHorizontal": 40,
      "paddingVertical": 20,
    }
  }
>
  <Text>
    #
  </Text>
</View>
`;
